src/store/sagas/app.ts

```diff
@@ -45,7 +44,6 @@ export function* updateAssets(action: RootAction) {
       );
       yield put(setAssets(assets));
       yield put(setAsset(assets[0]));
-      yield put(restartUpdateAlarms());
```

Comment:
is this change relevant?

Reply:
it's not relevant to widget save. but it's have to put there to working process correctly. (there was an error shown else)